android/build.gradle.kts

```
@@ -1,12 +1,10 @@
-import org.jetbrains.kotlin.konan.properties.loadProperties
-
 plugins {
     id(Plugins.ANDROID_APPLICATION)
     id(Plugins.KOVER)
     kotlin(Plugins.ANDROID)
 }
 
-val keystoreProperties = loadProperties("signing.properties")
+val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
```

Sorry, any reason why we revert this? 🙏 
I think I found the same issue [here](https://github.com/nimblehq/android-templates/pull/508/files#diff-2e3b887615bd52d5b6cea147228f1f8f684f0da8b6a6fb31323e54ec3856f5ccR12), and this is how to fix it

```suggestion
val signingProperties = loadProperties("$rootDir/signing.properties")
```

---

It's the same issue, let me use this solution 👍

---

I could not run the test cases but now I can with your solution @luongvo 👍